algorithms/kernel/dtrees/forest/classification/df_classification_training_types_result.h

```diff
@@ -34,6 +34,7 @@ namespace decision_forest
 {
 namespace training
 {
+services::Status checkImpl(const decision_forest::training::interface1::Parameter & prm);
```

Comment:
Some comment for latest parameter. It should be easier.

Reply:
Fixed